poetry.lock

```diff
@@ -278,24 +270,6 @@ category = "main"
 optional = false
 python-versions = "*"
 
-[[package]]
-name = "fastparquet"
-version = "0.7.2"
-description = "Python support for Parquet file format"
-category = "main"
-optional = false
-python-versions = ">=3.6,"
-
-[package.dependencies]
-cramjam = ">=2.3.0"
-fsspec = "*"
-numpy = ">=1.18"
-pandas = ">=1.1.0"
-thrift = ">=0.11.0"
```

Comment:
Interesting that fastparquet actually has more dependencies than pyarrow, which only has `numpy`.